docs/src/main/sphinx/connector/hudi.rst

```diff
@@ -0,0 +1,163 @@
+      - ``false``
+    * - ``hudi.columns-to-hide``
+      - List of column names that will be hidden from the query output.
+        It can be used to hide Hudi meta fields. By default, no fields are hidden.
+      -
+    * - ``hudi.parquet.use-column-names``
+      - Access Parquet columns using names from the file. If disabled, then columns
+        are accessed using index.Only applicable to Parquet file format.
+      - ``true``
+    * - ``hudi.min-partition-batch-size``
+      - Minimum number of partitions returned in a single batch.
+      - ``10``
+    * - ``hudi.max-partition-batch-size``
+      - Maximum number of partitions returned in a single batch.
+      - ``100``
+    * - ``hudi.size-based-split-weights-enabled``
+      - Unlike uniform splitting, size-based splitting ensures that each batch of splits
+        has enough data to process. By default, it is enabled to improve performance.
+      - ``true``
+    * - ``hudi.standard-split-weight-size``
+      - The split size corresponding to the standard weight (1.0)
+        when size based split weights are enabled.
+      - ``128MB``
+    * - ``hudi.minimum-assigned-split-weight``
+      - Minimum weight that a split can be assigned
+        when size based split weights are enabled.
+      - ``0.05``
+    * - ``hudi.max-splits-per-second``
+      - Rate at which splits are enqueued for processing.
+        The queue will throttle if this rate limit is breached.
+      - ``Integer.MAX_VALUE``
+    * - ``hudi.max-outstanding-splits``
+      - Maximum outstanding splits in a batch enqueued for processing.
+      - ``1000``
+
+Supported file types
+--------------------
+
+The following file types are supported for the Hudi connector:
+
+* Parquet
+
+SQL Support
+-----------
+
+Currently, the connector only provides read access to data in the Hudi table that has been synced to
+Hive metastore. Once the catalog has been configured as mentioned above, users can query the tables
+as usual like Hive tables.
+
+Supported Query Types
+^^^^^^^^^^^^^^^^^^^^^
+
+Hudi supports two types of tables depending on how the data is indexed and laid out on
+the file system. To learn more about table and query types in Hudi, please check
+`this doc <https://hudi.apache.org/docs/table_types>`_. Below table is the support matrix
+for Hudi connector.
+
+=========================== =============================================
+Table Type                  Supported Query types
+=========================== =============================================
+Copy On Write               Snapshot Queries
+
+Merge On Read               Read Optimized Queries
+=========================== =============================================
+
+Examples Queries
+^^^^^^^^^^^^^^^^
+
+``stock_ticks_cow`` is a Hudi COW table that we refer in the Hudi
+`quickstart <https://hudi.apache.org/docs/docker_demo/>`_ document to create.
+
+Here are some sample queries:
+
+.. code-block:: sql
+
+    USE hudi.default;
```

Comment:
Remove the hudi.default ... instead talk about the example catalog and the default schema